common/src/main/java/com/wynntils/models/characterstats/actionbar/ManaSegment.java

```diff
@@ -11,7 +11,8 @@
 import java.util.regex.Pattern;
 
 public class ManaSegment implements ActionBarSegment {
-    private static final Pattern MANA_PATTERN = Pattern.compile("§b✺ ([0-9]+)/([0-9]+)");
+    // https://regexr.com/7k9ab
+    private static final Pattern MANA_PATTERN = Pattern.compile("(?:§.  §.|§b {0,2})✺ (\\d+)/(\\d+)");
```

Comment:
I'm not sure what I am seeing here. Why do we get these prefixed spaces? Are we correct in pushing them on to the segment, or should we have removed them in the handler?

Reply:
This is just the raw result from styledtext.split() on the entire actionbar

Reply:
So the action bar looks something like: `§x<left>    §y<center>    §z   §w<right>`?

I also find the `|` without parenthesis in regexps confusing, since I think it binds strangely. Does not ` §.|§b ` mean "space, followed by either § and any character, or "§b" and then space"? If so, this could just as well be ` §. `. (And why could we have anything but the `b`?)

Reply:
I might be missing something here, but I think we might need to do some more processing "centrally" before sending out the segments to the segment handlers, and just not blindly split it. 

The other segment matchers looked okay, but this one does not.

Reply:
Or does `(?:§.  §.|§b {0,2})` mean either match `§.  §.` or `§b {0,2})`? If so, why is not the first part `§.  §b`? This does not seem to rhyme with the example string you posted in ActionBarHandler...

Reply:
I think this will be easier to discuss in Discord, I'll @ you